### ISSUE TITLE
I got it working with the chinese board

### ISSUE THREAD

**Issue by DrJaymz** [felis/USB_Host_Shield_2.0#810]
It didn't take that long but got it working with the chinese board you see around the place.  I found some references saying they had no luck but I pinged out the traces and everything you need was there.  There are various versions but here was mine.

![image](https://github.com/felis/USB_Host_Shield_2.0/assets/42401980/ada41313-8021-48e7-b78b-5ff2d1ef8faf)

I connected to ESP32 and it works fine.  I then enabled debug and plugged in a keyboard and loaded the keyboard demo and it detects and you can see the key presses.

My real goal was to use a Magnetic stripe reader to pull card numbers for a door entry system.  I can get it to work on one but not another.  Both are HID - sort of keyboard wedges, I had to use the multimedia keboard demo and then I could see the bytes and then decode them.  

But for the other device its Full Speed and I don't see it.  The MAX chip does support full-speed and I made sure that I provide enough 5V current, hence the mod to the board.  Using windows it comes up as a composite device with a keyboard and full speed but I have no idea how you'd code for this new device - since no OS requires drivers and sees as keyboard I hoped I'd made it.

**Comment by TheNathanSpace** [comment]
Hey, trying to follow in your footsteps here. What do you mean by the arrow from VCC 3.3 -> the spot by the cut trace with the label "connect"? (Sorry, probably don't have the exact right terminology.)

Did you actually solder in some wire connecting those two spots?

<img width="886" height="587" alt="Image" src="https://github.com/user-attachments/assets/b083f1e3-aaa8-4156-b90d-74fba6b4e834" />

**Comment by DrJaymz** [comment]
> Hey, trying to follow in your footsteps here. What do you mean by the arrow from VCC 3.3 -> the spot by the cut trace with the label "connect"? (Sorry, probably don't have the exact right terminology.)
> 
> Did you actually solder in some wire connecting those two spots?
> 
> <img alt="Image" width="886" height="587" src="https://private-user-images.githubusercontent.com/46632454/568135185-b083f1e3-aaa8-4156-b90d-74fba6b4e834.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzQzNjY5NTcsIm5iZiI6MTc3NDM2NjY1NywicGF0aCI6Ii80NjYzMjQ1NC81NjgxMzUxODUtYjA4M2YxZTMtYWFhOC00MTU2LWI5MGQtNzRmYmE2YjRlODM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAzMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMzI0VDE1MzczN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1Y2FjZGE3NWJkZjkyZTYxNjExYWY3ODEwMDRkMTBkNGM5ZTkxYmYyYTcxYWFjODQ1NTMzODBlYjkzNDMzMTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Je8ozeJAEdMMuuJyWU3m7bS6DSOQvK3zp2pKX36bQWU">

Yes, I did better than that - I created a product using that actual chip - so the issue was whoever created the board in the first place.

<img width="1213" height="765" alt="Image" src="https://github.com/user-attachments/assets/0cb4d1ef-ea0c-45b3-a747-9e6906bf0970" />

**Comment by TheNathanSpace** [comment]
Oh wow very cool! I haven't gotten my host header working yet, and I *feel like* my circuit design is correct but I'm just messing up the physical implementation. But seems like maybe it's a problem with the host header; serves me right for trying to save a few dollars. Oh well. Thanks for sharing your notes!

**Comment by DrJaymz** [comment]
> Oh wow very cool! I haven't gotten my host header working yet, and I _feel like_ my circuit design is correct but I'm just messing up the physical implementation. But seems like maybe it's a problem with the host header; serves me right for trying to save a few dollars. Oh well. Thanks for sharing your notes!

I can tell you that it can be made to work, I did.  And yes, you have to ensure its connected up as my diagram and cross reference schematic.
